src/test/run-make/wasm-abi/Makefile

```diff
@@ -0,0 +1,7 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# only-wasm32-bare
```

Comment:
Should this test eventually run for wasm64? Can you add a TODO comment to that effect?

Reply:
Indeed yeah there's nothing 32-bit specific about it, but afaik there's a big swath of the test suite that needs to be updated to not have "32" in the name of "only" or "ignore-wasm32", so I'm not sure it makes sense to only add a TODO here rather than handle it later when wasm64 is supported more